docs/reference/manifest/extendedpermissions.md

```diff
@@ -10,7 +10,9 @@ localization_priority: Normal
 Defines the collection of extended permissions the add-in needs to access associated APIs or features. The `ExtendedPermissions` element is a child element of [VersionOverrides](versionoverrides.md).
 
 > [!IMPORTANT]
-> This element is only available in the [Outlook add-ins preview requirement set](../objectmodel/preview-requirement-set/outlook-requirement-set-preview.md) against Exchange Online. Add-ins that use this element cannot be published to AppSource or deployed via centralized deployment.
+> Only Outlook on the web and Windows support the **ExtendedPermissions** element.
```

Comment:
FWIW, I find it confusing for a few seconds when we conjoin the object of the preposition "on" like this: "Outlook on the web and Windows". I think it would be smoother reading to either conjoin to prepositional phrases like this "Outlook on the web and on Windows" or, best of all, just conjoin two noun phrases like this: "Outlook on the web and Outlook on Windows"